operator/standalone/create.go

```diff
@@ -291,6 +306,74 @@ func (p *CreateStandalonePipeline) markInstanceAsReady(ctx context.Context) erro
 	return p.client.Status().Update(ctx, p.instance)
 }
 
+// ensureConnectionSecret creates the connection secret in the instance's namespace.
+func (p *CreateStandalonePipeline) ensureConnectionSecret(ctx context.Context) error {
+	err := Upsert(ctx, getClientFromContext(ctx), p.connectionSecret)
+	return err
+}
+
+// fetchCredentialSecret gets the credential secret and puts the credentials for postgresql into the connection secret.
+func (p *CreateStandalonePipeline) fetchCredentialSecret(ctx context.Context) error {
+	secret := &corev1.Secret{ObjectMeta: metav1.ObjectMeta{
+		Name:      getCredentialSecretName(),
+		Namespace: p.instance.Status.HelmChart.DeploymentNamespace,
+	}}
+	err := getClientFromContext(ctx).Get(ctx, types.NamespacedName{Name: secret.Name, Namespace: secret.Namespace}, secret)
+	if err != nil {
+		return err
+	}
+	if p.instance.Spec.Parameters.EnableSuperUser {
```

Comment:
Currently `EnableSuperUser` can be changed during update, we have to keep in mind to either make it immutable or update the code where necessary

Reply:
Updates are out of scope at this time. Let's change when it's necessary.